docs/Agent/installation/ansible_based_installation.md

```diff
@@ -0,0 +1,37 @@
+# Ansible based installation
+
+In the following, we describe how to install Pbench Agent using an ansible playbook.
+
+:::{note}
+The same Pbench Agent version must be installed on all the test systems that participate in a benchmark run, there is no support for mixed installations.
+:::
+
+## Setup
+
+1. Make sure that you have ansible package installed.
+
+2. Install the pbench.agent ansible collection from Ansible Galaxy.
```

Comment:
"pbench.agent" is literal text, so it should probably be in a monospace font.  (And, "ansible" should probably be capitalized...ditto line 19.)